app/src/renderer/components/common/AppMenu.vue

```diff
@@ -3,7 +3,10 @@ menu.app-menu
   .app-menu-main
     list-item(to="/" exact @click.native="close" title="Wallet")
     list-item(to="/wallet/transactions" exact @click.native="close" title="Transactions" v-if="config.devMode")
-    list-item(to="/staking" exact @click.native="close" title="Staking" v-bind:class="{ 'active': isValidatorPage }" v-if="config.devMode")
+
+    // Hide staking for now (see https://github.com/cosmos/voyager/issues/720).
+    //- list-item(to="/staking" exact @click.native="close" title="Staking" v-bind:class="{ 'active': isValidatorPage }" v-if="config.devMode")
```

Comment:
we have a development mode flag `v-if="config.devMode"` that is already applied. If the staking option is still visible, the user has `NODE_ENV=development` set globally.